docker-compose.yml

```
@@ -1,28 +1,25 @@
 version: '3.6'
-
 services:
   prem_app:
     container_name: prem_app
     build: .
     environment:
-      - VITE_DESTINATION=browser
-      - VITE_IS_PACKAGED=true
-  
+    - VITE_DESTINATION=browser
+    - VITE_IS_PACKAGED=true
   premd:
     container_name: premd
     image: ghcr.io/premai-io/premd:v0.0.20
```

should this be changed?

```suggestion
    image: ghcr.io/premai-io/premd:v0.0.22
```

```suggestion
    image: ghcr.io/premai-io/premd:latest
```